REQUIRE

```diff
@@ -4,7 +4,7 @@ Distributions
 Docile
 DualNumbers
 FITSIO
-ForwardDiff 0.0.2 0.0.3
+ForwardDiff 0.0.2 0.0.3+
```

Comment:
The current release is only compatible with Julia 4.0.  I asked them to tag 0.0.3 which has the support for the typed lgamma hessians that I added.  I haven't been following the stability of 4.0 much, but from the last pull request it looks like maybe you have.  Any thoughts on when we should switch?